### DISCUSSION TITLE
Victron Lastausgag schalten

### DISCUSSION THREAD

**Discussion by ar12321** [hoylabs/OpenDTU-OnBattery#250] [General]
Hallo,
kann man über opendtu auch den Lastausgang eines Victron Ladereglers schalten?

**Comment by madmartin** [comment]
Nein, das ist nicht möglich.

**Comment by ar12321** [comment]
Danke für die Antwort.
Ist es denkbar, mit dem Projekt einen Microprozessorausgang schaltbar zu gestalten?
Darüber könnte z.B. über eine externe Steuerung der Lastausgang eines Victron geschaltet werden.
Die Ansteuerung könnte wie die externe Leistungssteuerung zum Hoymiles erfolgen.

**Reply by madmartin** [reply]
> Ist es denkbar, mit dem Projekt einen Microprozessorausgang schaltbar zu gestalten? 

Ja klar. 

Ich habe für meinen SmartSolar 100/20 eben in der VictronConnect App mal die Einstellungen für den RX Port angesehen. Da gibt es die Einstellung "Lastausgang an/aus normal" (und invertiert). 
Wie das vom ESP angesteuert werden kann (ist eigentlich als serieller Port konfiguriert) weiß ich nicht - hab ich hier noch niemand implementieren sehen.

> Die Ansteuerung könnte wie die externe Leistungssteuerung zum Hoymiles erfolgen.

Nach welchen Kriterien soll denn da geschaltet werden?
Die "Leistungssteuerung" des Hoymiles Wechselrichter erfolgt nach %-werten und - bei 0% - wird auch ausgeschaltet. 
Was willst Du da mit einem "an/aus" Schalter abbilden?

**Reply by ar12321** [reply]
Hallo,
ich habe auch den SmartSolar 100/20. Ich schalte den Lastausgang über die Batteriespannung (Voreinstellung). Gern würde ich auch über externe Steuerung (FHEM) schalten, wenn z.B. der Tagesertrag nicht ganz reicht. (FHEM MQTT-Befehl, z.B. abends bei Batterie >80% einschalten).
Zum Teil macht es auch Sinn bei voller Einstrahlung den SS100/20 zuzuschalten, so dass alle PV-Module voll genutzt werden (dann 600W über HM600 + 350W Batterieladung).
Der Power-Limiter funktioniert bei meiner Konfiguration leider nicht, so dass ich über FHEM bei Batteriebetrieb lastabhängig einen Zweistufenbetrieb des HM600 steuere.
Vielleicht könnte man dem Power-Limiter auch Schaltflächen für konfigurationsabhängige Leistungsregelung hinzufügen

Meine Konfiguration ist:  HM600 am Stromnetz
- HM600-Eingang 1:  PV 395kWp
- HM600 Eingang 2:  36V/50Ah Batterie, über Smartsolar 100/20 mit 2xPV 395kWp

Noch eine Frage, gibt es die Möglichkeit die Gesamtanschlussleistung (Leistung Netz) wie in der OpenDTU-App per MQTT zu übertragen?
